src/tool/pmix_tool.c

```diff
@@ -644,6 +644,7 @@ PMIX_EXPORT int PMIx_tool_init(pmix_proc_t *proc, pmix_info_t info[], size_t nin
     pmix_globals.iof_flags.local_output = outputio;
 
     /* setup the globals */
+    PMIX_CONSTRUCT(&pmix_client_globals.groups, pmix_list_t);
```

Comment:
Need to add the corresponding `PMIX_LIST_DESTRUCT` to the tool's "finalize" routine